src/basic/terminal-util.c

```diff
@@ -1204,7 +1204,7 @@ bool terminal_is_dumb(void) {
         return streq(e, "dumb");
 }
```

Comment:
i think it would be nicer to add a new static function "getenv_term_is_dumb()" or so, that just checks $TERM, and then leave terminal_is_dumb() without any argument, and make it call this new function. The existing code could then continue to use terminal_is_dumb() as before, except for colors_enabled(), which would be changed to use getenv_term_is_dumb() instead of terminal_is_dumb()...